### DISCUSSION TITLE
MySQL option

### DISCUSSION THREAD

**Discussion by sdenike** [CyferShepard/Jellystat#164] [Ideas]
I would love to see an option to use an already up and running MySQL server so that postgres was not the only option.  Would this be possible on the roadmap?

**Comment by CyferShepard** [comment]
Hi @sdenike , this is not on the roadmap as it would require an extensive rewrite of jellystat. Im using postgres for the extensive sql functionality it provides which allows for faster data handling. If i had to use another db most of the functions on the database side would have to be rewritten in the application instead and this would greatly reduce performance.

**Reply by sdenike** [reply]
Thanks for the update on this and totally understandable, I figured I would ask as I wasn't sure if it was modular in design and using a database connector that could be changed from X to Y easy.  Thanks again!
